### DISCUSSION TITLE
Custom Additions to Model Builder Class/ Custom Control Objects

### DISCUSSION THREAD

**Discussion by Sina-Haz** [NVIDIA/warp#320] [General]
Hi, I notice the Model/Model Builder class doesn't implement anything like cables that are connected on a point to rigid bodies. Is there a way to customly define a struct for cables and a custom control object being changing its rest lengths and then integrate that with the Model or ModelBuilder class?

**Comment by eric-heiden** [comment]
Continuing the conversation in https://github.com/NVIDIA/warp/issues/330
